Demo/Tests/SnapshotTest/SnapshotTest.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  File.swift
+//  Tests
+//
+//  Created by rtysiachnik on 11/1/17.
+//  Copyright © 2017 One by AOL : Publishers. All rights reserved.
+//
+//
+import FBSnapshotTestCase
+import UIKit
+
+open class SnapshotTest: FBSnapshotTestCase {
+    
+    override open func setUp() {
+        super.setUp()
+        recordMode = nil != ProcessInfo.processInfo.environment["RECORD_MODE"]
```

Comment:
What is the purpose of this variable?

Reply:
This variable will have another design in upcoming commit. However, it's purpose is the same. It is to turn on/off the record mode for all of these tests. The value comes out of fastfile and it is used as xcarg in scan action.